TeamViewerPS/Public/Add-TeamViewerManager.ps1

```diff
@@ -77,7 +77,7 @@ function Add-TeamViewerManager {
             $body["id"] = $Manager | Resolve-TeamViewerManagerId
         }
         '*ByUserObject' {
-            $body["accountId"] = $User.Id.TrimStart('u')
+            $body["accountId"] = $User.TrimStart('u')
```

Comment:
I think this wasn't correct in the first place - and also isn't quite correct now.
Could be replaced by something like this:

```powershell
$body["accountId"] = ($User | Resolve-TeamViewerUserId).TrimStart('u')
```